server/routes/users.js

```diff
@@ -1,13 +1,54 @@
-const express = require('express');
+const express = require("express");
 const router = express.Router();
 
+let Event = require("../models/event.model");
+let User = require("../models/user.model");
+
 /* GET users listing. */
-router.get('/', function(req, res, next) {
+router.get("/", function (req, res, next) {
   const user = {
-    name: 'ACM Hack',
-    email: 'hack@acmucsd.org'
-  }
+    name: "ACM Hack",
+    email: "hack@acmucsd.org",
+  };
   res.status(200).json({ user });
 });
 
+router.put("/events/:id", function (req, res) {
+  try {
+    const { id, update } = req.params;
```

Comment:
Since this function will be performing asynchronous operations (e.g. with `findByIdAndUpdate`), it needs to be marked as `async`.
```suggestion
router.put("/events/:id", async function (req, res) {
  try {
    const { id, update } = req.params;
```